packages/paste-website/src/pages/components/callout/index.mdx

```diff
@@ -0,0 +1,236 @@
+  </Paragraph>
+  <Callout variant='neutral' marginY='space70'>
+    <CalloutHeading as='h3'>Accessibility insight</CalloutHeading>
+    <CalloutText>Make sure every Input is associated with a Label.</CalloutText>
+  </Callout>
+  <Paragraph>
+    Mi in nulla posuere sollicitudin aliquam ultrices. Consectetur libero id faucibus nisl tincidunt eget nullam non. Imperdiet nulla malesuada pellentesque elit eget. Est ante in nibh mauris cursus mattis molestie. Vitae proin sagittis nisl rhoncus mattis rhoncus urna neque viverra. Phasellus vestibulum lorem sed risus ultricies tristique. Dui vivamus arcu felis bibendum ut.
+  </Paragraph>
+</>`}
+</LivePreview>
+
+## Usage Guide
+
+### API
+
+#### Installation
+
+```bash
+yarn add @twilio-paste/callout - or - yarn add @twilio-paste/core
+```
+
+#### Usage
+
+```jsx
+import {Callout, CalloutHeading, CalloutText} from '@twilio-paste/core/callout';
+
+const ExampleCallout = () => (
+  <Callout variant="neutral">
+    <CalloutHeading as="h3">Heads up!</CalloutHeading>
+    <CalloutText>This is some information you need to know.</CalloutText>
+  </Callout>
+);
+```
+
+#### Props
+
+##### Callout
+
+| Prop      | Type                                                         | Description                                                                               | Default   |
+| --------- | ------------------------------------------------------------ | ----------------------------------------------------------------------------------------- | --------- |
+| children? | `React.ReactNode`                                            | Child components to render into the card                                                  |           |
+| variant   | `'neutral', 'warning', 'error', 'success', 'new', 'default'` | The variant of Callout to render                                                          |           |
+| marginY   | `ResponsiveValue<keyof ThemeShape['space']>`                 | Any html attrs like aria, data etc.                                                       |           |
+| element?  | `string`                                                     | Overrides the default element name to apply unique styles with the Customization Provider | 'CALLOUT' |
+
+##### CalloutHeading
+
+| Prop      | Type                                                                   | Description                                                                               | Default           |
+| --------- | ---------------------------------------------------------------------- | ----------------------------------------------------------------------------------------- | ----------------- |
+| children? | `React.ReactNode`                                                      | Child components to render into the card                                                  |                   |
```

Comment:
suggestion: you could also change all these to "into the _component_" for easier copy and pasting
```suggestion
| children? | `React.ReactNode`                                                      | Child components to render into the heading                                                  |                   |
```